lib/backpex/fields/select.ex

```diff
@@ -25,7 +25,7 @@ defmodule Backpex.Fields.Select do
   @impl Backpex.Field
   def render_value(assigns) do
     options = Map.get(assigns.field_options, :options)
```

Comment:
```suggestion
    options = get_options(assigns.field_options)
```

In my opinion, it would be better to get the options here. We could change `assigns_options` to `get_options`. Then we do not need to change the logic of `get_label` and can leave the rest as it is.